sdk/quantum/quantum-jobs/api-extractor.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
-  "mainEntryPointFilePath": "./types/index.d.ts",
+  "mainEntryPointFilePath": "./types/src/index.d.ts",
   "docModel": { "enabled": true },
   "apiReport": { "enabled": true, "reportFolder": "./review" },
   "dtsRollup": {
```

Comment:
Update 
`"publicTrimmedFilePath": "./esm/index.d.ts"`
to
`"publicTrimmedFilePath": "./types/latest/quantum-jobs.d.ts"`

And please make sure to link it in the package.json with the `"types"` property.

Reply:
Same as above